addon/components/influencer-network-modal.ts

```diff
@@ -0,0 +1,61 @@
+import Component from '@glimmer/component';
+import UPFLocalStorage from '@upfluence/oss-components/utils/upf-local-storage'
+import { tracked } from '@glimmer/tracking';
+import { GLOBAL_SUPPORT_LINK } from '@upfluence/ember-upf-utils/resources/helpdesk-links';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+const INFLUENCER_NETWORK_MODAL_COOKIE = 'upf_disable_influencer_modal';
+const INFLUENCER_NETWORK_MODAL_STORAGE_KEY = 'disable_influencer_modal';
+
+interface ComponentSignature {
+  hideInfluencerNetworkModal: boolean;
+  onClose(): void;
+}
+
+export default class extends Component<ComponentSignature> {
+  @service declare currentUser: any;
```

Comment:
Use `any` instead of currentUserService from `@upfluence/ember-identity` to avoid adding a new dependency